### DISCUSSION TITLE
Json aggregate functions

### DISCUSSION THREAD

**Discussion by baszalmstra** [SeaQL/sea-query#442] [General]
Is there support planned for json aggregate functions like Postgres’ json_agg/row_to_json? Would be very useful for building optimized queries for GraphQL

**Comment by ikrivosheev** [comment]
@baszalmstra yes, we can add support for the json aggregate function. I think after adding support for the Postgres array. 
Can you create issue?)

**Reply by baszalmstra** [reply]
See: https://github.com/SeaQL/sea-query/issues/463

**Reply by ikrivosheev** [reply]
Thank you!
